src/modules/animations/led-hover/ledImpactAnim.js

```diff
@@ -27,7 +29,7 @@ ledPlaneScaleAnim.setAttribute('direction', 'alternate');
 ledPlaneScaleAnim.setAttribute('ease', 'ease-in-out');
 let ledPlaneScaleAnimActive = true;
 
-function show(selectors, helpers) {
+function show(selectors) {
 	document.querySelector('#led-text').setAttribute('opacity', '0');
 	document.querySelector('#led-sub-text').setAttribute('opacity', '0');
 	document.querySelector('#led-text').setAttribute('position', '0 0 0');
```

Comment:
Could clean this up by binding element to a variable, and then do multiple operations on the same variable - element is a reference.

This will have the same effect, but be much easier to maintain:
```js
const ledTextElement = document.querySelector('#led-text');
ledTextElement.setAttribute('opacity', '0');
ledTextElement.setAttribute('position', '0 0 0');
```